doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -735,6 +735,14 @@ Once a dataset with embargoed files has been published, no further action is nee
 
 As the primary use case of embargoes is to make the existence of data known now, with a promise (to a journal, project team, etc.) that the data itself will become available at a given future date, users cannot change an embargo once a dataset version is published. Dataverse instance administrators do have the ability to correct mistakes and make changes if/when circumstances warrant.
 
+Retentions
```

Comment:
"Retention" isn't as common a term as "embargo" so I'd suggest changing the Guides info to talk about "retention period". If you want, I can either make suggestions here or make a PR against your branch with changes.

Reply:
I will try to change it, and some extra eyeballing is much appreciated

Reply:
@qqmyers Do we need to change 'Retention Expired" to "Retention Period Expired" in all GUI strings as well?

Reply:
Yeah, I think that would be better.

Reply:
@qqmyers There are still  some lines that I am no sure about; when we have 'under retention' as in "Is under retention until", should that be changed to 'retained'?

Reply:
Changed it to 'retained' seems best